### PR TITLE
Bug Fix

### LINKED ISSUE #204: [PE-D] Incorrect error message for menu edit
Suppose 5 items in Menu and user uses menu edit on index out of bounds, the error message shown should refer to "menu index" instead of "person index"

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: jxrrelo/ped#3

### LINKED ISSUE #205: [PE-D] Menu edit command on multiple similar fields
Command used: menu edit 5 n/Fries1 n/Fries2 p/5 p/6

For multiple inputs of p/5 p/6, app just accepts the latest argument p/6 and drops p/5. Should the app show an error in command format instead?

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: jxrrelo/ped#4

### PR DESCRIPTION
**Pull request by Sharptail**
Fix #204 
Fix #205

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -88,8 +88,22 @@ component command /prefix [input] -flag
...
+  The command above will execute `customer list` and ignore `JJIMY is the best!`
+  <p>&nbsp;</p>
+  
+- Only the latest input for any duplicated prefixes will be taken, any input for previous duplicated prefixes will be ignored.  
+  Examples:
+  ```
+  menu edit 5 n/Fries1 n/Fries2 p/5 p/6
+  ```
+  The command above will only take in `n/Fries2` and `p/6`, any other input for the same prefixes will be ignored.  
+  <p>&nbsp;</p>
```
Comment (guanyz) [inline docs/UserGuide.md]: This bit might look better in an infobox rather than a normal paragraph, I think?

Comment (Sharptail) [reply docs/UserGuide.md]: Good idea! resolving this in the next commit.

**Review by guanyz — APPROVED** [review]
LGTM!

**Review by IanCKW — APPROVED** [review]
Lgtm

**Pull request merged**